### DISCUSSION TITLE
UVtools gets crazy numbers for 2nd lift speed and height [Bug]

### DISCUSSION THREAD

**Discussion by aimmet** [sn4k3/UVtools#629] [Q&A]
### System

```Text
UVtools v3.9.3 X64
Operative system: Microsoft Windows 10.0.19044 X64
Processor: 12th Gen Intel(R) Core(TM) i9-12900K
Processor cores: 24
Memory RAM: 15.99 / 31.75 GB
Runtime: win10-x64
Framework: .NET 6.0.12
AvaloniaUI: 0.10.18
OpenCV: 4.6.0

Sreens, resolution, working area, usable area:
1: 2560 x 1080 @ 100% (Primary) (On this)
    WA: 2560 x 1040    UA: 2560 x 1040


Path:       C:\Users\Amier\Pictures\UVtools_win-x64_v3.9.3\
Executable: C:\Users\Amier\Pictures\UVtools_win-x64_v3.9.3\UVtools.exe
Loaded file: testforFORGE.osf [Version: 4] [Class: OSFFile]
```

### Printer and Slicer

- Printer:  Phenom Forge
- Slicer: lychee

### Description of the bug

![Error](https://user-images.githubusercontent.com/26447048/208786250-a4127dee-b757-4a60-9eff-0d26bc5f3924.PNG)

### How to reproduce

when i go to edit print parameters
Will this mess up the sliced file if i export it? Do I just have to reenter the proper credentials for 2nd lift?

**Comment by sn4k3** [comment]
UVtools only shows data when loading. This might be related to lychee and should be reported with them.

> Will this mess up the sliced file if i export it? 

I have no idea, but the values should be clamp in firmware to a limit.
If you want send the raw file so I can check, but I belive it come like that from lychee.
This format is messed up
